discojs/discojs-core/src/tasks/geotags.ts

```diff
@@ -0,0 +1,60 @@
+import { tf, Task } from '..'
+import { ImagePreprocessing } from '../dataset/data/preprocessing'
+
+export const task: Task = {
+  taskID: 'geotags',
+  displayInformation: {
+    taskTitle: 'GeoTags',
+    summary: {
+      preview: 'In this challenge, we predict the geo-location of a photo given its pixels in terms of a cell number of a grid built on top of Switzerland',
+      overview: 'The geotags dataset is a collection of images with geo-location information used to train a machine learning algorithm to predict the location of a photo given its pixels.'
+    },
+    limitations: 'The training data is limited to images of size 224x224.',
+    tradeoffs: 'Training success strongly depends on label distribution',
+    dataFormatInformation: 'Images should be of .png format and of size 224x224. <br> The label file should be .csv, where each row contains a file_name, class.  The class is the cell number of a the given grid of Switzerland. '
+  },
+  trainingInformation: {
+    modelID: 'geotags-model',
+    epochs: 10,
+    roundDuration: 10,
+    validationSplit: 0.2,
+    batchSize: 10,
+    modelCompileData: {
+      optimizer: 'adam',
+      loss: 'categoricalCrossentropy',
+      metrics: ['accuracy']
+    },
+    dataType: 'image',
+    preprocessingFunctions: [ImagePreprocessing.Resize],
+    LABEL_LIST: Array.from(Array(140).keys()).map(String),
```

Comment:
perfect, but maybe `immutable:Range` can help here

```suggestion
    LABEL_LIST: Range(0, 140).map(String).toArray(),
```

Reply:
Are you sure that Range exists?